lib/constants/activityTypes.js

```diff
@@ -0,0 +1,14 @@
+export default {
```

Comment:
Is that file used somewhere?

Reply:
Ops, this is a leftover from my previous commit. I've removed this file. 👍🏽